packages/tupaia-web/src/features/Dashboard/Dashboard.tsx

```diff
@@ -209,11 +213,11 @@ export const Dashboard = () => {
           onClose={() => setExportModalOpen(false)}
           dashboardItems={activeDashboard?.items as DashboardItemType[]}
         />
-        <SubscribeModal 
-            isOpen={subscribeModalOpen}
-            onClose={() => setSubscribeModalOpen(false)}
-            activeDashboard={activeDashboard}
-            onSubscribe={refetch}
+        <SubscribeModal
+          isOpen={subscribeModalOpen}
+          onClose={() => setSubscribeModalOpen(false)}
+          activeDashboard={activeDashboard}
+          onSubscriptionRequestSuccess={refetch}
```

Comment:
```suggestion
          onToggleSubscription={refetch}
```
Reads a little more obviously to me.